inhouse/command_handlers/soloqueue_leaderboard.py

```diff
@@ -0,0 +1,134 @@
+import discord
+from operator import itemgetter
+
+class Soloqueue_Leaderboard(object):
+    def __init__(self) -> None:
+        self.challenger_dict = {"I":[]}
+        self.grandmaster_dict = {"I":[]}
+        self.master_dict = {"I":[]}
+        self.diamond_dict = {"I":[],"II":[],"III":[],"IV":[]}
+        self.platinum_dict = {"I":[],"II":[],"III":[],"IV":[]}
+        self.gold_dict = {"I":[],"II":[],"III":[],"IV":[]}
+        self.silver_dict = {"I":[],"II":[],"III":[],"IV":[]}
+        self.bronze_dict = {"I":[],"II":[],"III":[],"IV":[]}
+        self.iron_dict = {"I":[],"II":[],"III":[],"IV":[]}
+        self.unranked_dict = []
+    def add_player(self,name,tier,rank,lp):
+        if tier == "CHALLENGER":
+            self.challenger_dict[rank].append((name, lp))
+        elif tier == "GRANDMASTER":
+            self.grandmaster_dict[rank].append((name, lp))
+        elif tier == "MASTER":
+            self.master_dict[rank].append((name, lp))
+        elif tier == "DIAMOND":
+            self.diamond_dict[rank].append((name, lp))
+        elif tier == "PLATINUM":
+            self.platinum_dict[rank].append((name, lp))
+        elif tier == "GOLD":
+            self.gold_dict[rank].append((name, lp))
+        elif tier == "SILVER":
+            self.silver_dict[rank].append((name, lp))
+        elif tier == "BRONZE":
+            self.bronze_dict[rank].append((name, lp))
+        elif tier == "IRON":
+            self.iron_dict[rank].append((name, lp))
+        else:
+            self.unranked_dict.append((name, lp))
+    def get_embbeded(self):
+        msg_list = []
+        i = 0
+        total_string = "```Soloqueue Leaderboard```\n"
+        msg = None
+        for rank in self.challenger_dict:
+            for player in sorted(self.challenger_dict[rank], key = itemgetter(1), reverse=True):
+                total_string += player[0] + ": " + "Challenger " + rank + " " + str(player[1]) + "LP\n"
```

Comment:
\>:( string interpolation!!!!
```suggestion
total_string += f"{player[0]}: Challenger {rank} {str(player[1])} LP\n"
```